bindings/python/htcondor2/_collector.py

```diff
@@ -39,9 +40,9 @@ class Collector():
     # In version 1, there was a distinct DaemonLocation type (a named tuple)
     # that `pool` could also be, but that functionality was never documented.
     #
-    def __init__(self, pool : Optional[Union[str, classad.ClassAd, List[str]]] = None):
+    def __init__(self, pool : Optional[Union[str, classad.ClassAd, List[str], Tuple[str]]] = None):
```

Comment:
```suggestion
    def __init__(self, pool : Union[str, classad.ClassAd, List[str], Tuple[str], None] = None):
```
Optional[foo] is syntactic sugar for Union[foo, None]. Since you're already using Union, you can get rid of a level of nesting here.